.github/workflows/valgrind.yml

```diff
@@ -0,0 +1,42 @@
+name: valgrind leak check
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest 
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: install packages
+      run: |
+        sudo apt-get install -y libavcodec-dev \
+                            libavdevice-dev \
+                            libavformat-dev \
+                            libgtk2.0-dev \
+                            libjack-jackd2-dev \
+                            libmpg123-dev \
+                            libopus-dev \
+                            libpulse-dev \
+                            libsndfile1-dev \
+                            libspandsp-dev \
+                            libssl-dev \
+                            libglib2.0-dev \
+                            valgrind
```

Comment:
selftest only depends on libbaresip and no modules.
perhaps some -dev packages can be skipped ?